docs/button.rst

```diff
@@ -0,0 +1,36 @@
+Buttons
+*******
+
+.. py::module:: microbit
+
+There are two buttons on the board, called ``button_a`` and ``button_b``.
+
+
+Classes
```

Comment:
Not sure if this is the correct thing to do, but suggesting it all the same for the sake of something to discuss. We should remove the Class section and replace it with something along the lines of:

```
.. py:attribute:: button_a

    Represents button A.

    .. py:method:: is_pressed()

        Returns ``True`` if the button is pressed, and ``False`` otherwise.
```

...etc...